.github/workflows/Ci-raiwigets-python-typescript.yml

```diff
@@ -108,8 +108,7 @@ jobs:
         working-directory: ${{ matrix.packageDirectory }}
 
       # Only try to upload code cov if python tests were run
-      - if: ${{ (steps.raiwidgettests.outcome == 'success') }}
-        name: Upload code coverage results
+      - name: Upload code coverage results
```

Comment:
This lives in CI-raiwidgets-pytest.yml now since my PR merged.

Reply:
I am going to revert all these changes after testing. There were weird issues where the E2E screenshots are not get uploaded.